### DISCUSSION TITLE
Java exception UnknownHostException

### DISCUSSION THREAD

**Discussion by rudyslee** [iTwin/itwinjs-core#4930] [Q&A]
Hi,

I am new to iTwin and I use Java to call the API and I encountered this error below when try to get access token;
> javax.ws.rs.ProcessingException: java.net.UnknownHostException: UnknownHostException invoking https://ims.bentley.com/connect/token: ims.bentley.com

It seems the computer where I executed the Java program was unable to resolve ims.bentley.com.

Please advise how to solve this issue.

Thanks & Regards,
Rudy

**Comment by arnab-git-hub** [comment]
Hi,

Can you please reproduce the issue without using java? We usually use C# or typescript to get the IMS token.
